lib/error-matcher.js

```diff
@@ -23,12 +23,21 @@ module.exports = (function () {
   util.inherits(ErrorMatcher, EventEmitter);
 
   ErrorMatcher.prototype._goto = function () {
-    var file = this.cwd + path.sep + this.currentMatch[0].file;
-    if (!this.currentMatch[0].file) {
+    var file = this.currentMatch[0].file;
+    if (file) {
+      var oldDir = process.cwd();
+      process.chdir(this.cwd);
```

Comment:
I agree that we should look to deprecate `absFile` in regard to this change.

I still don't see why you need to `chdir`. Check out [these changes I pushed to a dev branch](https://github.com/noseglid/atom-build/commit/a452263491dee7e9d8b93ef04b370ddf8a188cf6). Wouldn't that do it?